sqs_workers/processors.py

```diff
@@ -59,11 +59,49 @@ def process_message(self, message):
         else:
             return True
 
+    def process_messages(self, messages):
+        """
+        Take a list of messages and send them in a batch
+        to the decorated call handler function.
+        """
+        message_ids = [m.message_id for m in messages]
+        extra = {
+            "message_ids": message_ids,
+            "queue_name": self.queue.name,
+            "job_name": self.job_name,
+        }
+        logger.debug(
+            "Processing batch for {queue_name}.{job_name}".format(**extra), extra=extra
+        )
```

Comment:
It's bad practice to format log strings: keeping "%s" etc. in them allows logging systems like Sentry or DataDog to group similar messages.
```suggestion
        logger.debug(
            "Processing batch for %s.%s",
            self.queue_name,
            self.job_name,
             extra=extra,
        )
```

Reply:
Done, this style of logging was scattered across the project so adjusted them all now to use this type of interpolation.